src/DGtal/shapes/TriangulatedSurface.h

```diff
@@ -546,6 +550,77 @@ namespace DGtal
     */
     VertexRange allBoundaryVertices() const;
 
+    /**
+       @param a any arc.
+       @return either the four vertices of the two triangles bordering \a a, or the three vertices if this is an edge on the boundary.
+       @note O(1) operation
+       @note Order is, if arc a is (s,t), and a belongs to triangle (s,t,u) and opposite arc belongs to triangle (t,s,v): (t,u,s,v), (t,u,s), (t,s,v) 
+    */
+    VertexRange verticesAroundArc( const Arc a ) const;
```

Comment:
```suggestion
    VertexRange verticesOfFacesAroundArc( const Arc a ) const;
```
This is a bit overloaded but less confusing that "verticesAroundArc" which would be read as "(the two) vertices of an arc"

Reply:
(head(a) tail(a) metthods)

Reply:
Ok. I've changed as you said.